packit_service/models.py

```diff
@@ -779,6 +779,7 @@ class CoprBuildModel(ProjectAndTriggersConnector, Base):
     web_url = Column(String)
     # url to copr build logs
     build_logs_url = Column(String)
+    task_accepted_time = Column(DateTime)
```

Comment:
nit: please provide a comment what the value is meant to contain, I'm assuming it's something like

    # time when we receive events and create respective copr builds